resource-metadata/hypothesis.md

```diff
@@ -0,0 +1,121 @@
+# Resource Metadata Model
+
+## Background
+
+This document introduces a mechanism for linking to metadata about resources in the Solid Ecosystem using HTTP Link headers. Examples of this linking mechanism include:
+
+- A binary JPEG image with a Link header, pointing to a description that has an RDF representation.
```

Comment:
I have created dedicated issue for media resources and multiple encodings of same creative work #33 . Here I would like to clarify if we talk about description of each of those encodings or description of the creative work which can have numerous encodings?

Reply:
If we just talk about different representation of the same resource, I think we could just use `rel=alternate` together with `type=image/jpeg` http://microformats.org/wiki/rel-alternate#With_type

> When used in combination with the `type` attribute (with a value other than that of the document itself; e.g. other than `"text/html"`), `rel="alternate"` means a link to a representation of the contents of the current document in a different format, as designated by the `type` attribute. 

If we talk about relating a creative work to multiple different encodings, each of which can have their own encoding specific rdf representation, i think should discuss it in #33

Reply:
Would you say that the proposed approach we describe here gives us the basic capabilities needed to make these associations (for example in the content of the resource description metadata resource)?